### DISCUSSION TITLE
Mac版本的client如何能够使用http代理

### DISCUSSION THREAD

**Discussion by myahmao** [1715173329/UnblockNeteaseMusic#863] [Q&A]
客户端版本是1.5.9
从big sur升级到Monterey之前，http代理是可以用的。
升级了以后代理就不能用了。
错误提示

`neteasemusic - music.163.com:443 HTTP connection error: Client send incorrect HTTP request. Please make sure that client uses HTTP  protocol.`

如何能迫使客户端使用http而不是https，或者有别的什么办法没有？

**Comment by gavin8282** [comment]
使用pac代理。一直正常

**Comment by 1715173329** [comment]
我们已经支持 https 代理，请正确配置。

**Comment by myahmao** [comment]
如何正确配置https代理？
$node app.js -a 0.0.0.0 -p 8081:8082 -o youtube kuwo kugou qq joox -s
INFO: (app) HTTP Server running @ http://0.0.0.0:8081
INFO: (app) HTTPS Server running @ http://0.0.0.0:8082

pac设置成端口8081或者8082都不管用
还需要配置证书吗？

**Reply by 1715173329** [reply]
pac 使用 8081，目标设备必须信任项目证书。
